evadb/binder/statement_binder.py

```diff
@@ -199,6 +200,9 @@ def _bind_select_statement(self, node: SelectStatement):
                 node.target_list = extend_star(self._binder_context)
             for expr in node.target_list:
                 self.bind(expr)
+                if isinstance(expr, FunctionExpression):
+                    add_func_expr_outputs_to_binder_context(expr, self._binder_context)
```

Comment:
How about other expressions?

Reply:
Only function expressions generate new columns. 
We don't support `AS` in the projection list. Will extend the support after doing it.